index.html

```diff
@@ -1,70 +1,218 @@
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
 
-  </div>
+      <div class="bg-card">
+        <img src="images/mps-1.png" alt="MPS project seakpeak">
+        <h1>Multi-Post 
+          <br>Stories</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
 
-  <div class="footer">
-    <h3>LET'S CONNECT</h3>
-  <ul class="icon-bar">
-    <li><a href="https://twitter.com/87juda">
-      <img src="images/twitter.png" alt="Twitter">
-    </a></li>
-    <li><a href="https://www.linkedin.com/in/judateixeira/">
-      <img src="images/linkedin.png" alt="Linkedin">
-    </a></li>
-    <li><a href="https://web.facebook.com/judateixeira/">
-      <img src="images/facebook.png" alt="Facebook">
-    </a></li>
-    <li><a href="https://github.com/mrjuda">
-      <img src="images/github.png" alt="Github">
-    </a></li>
-    <li><a href="https://instagram.com/srjuda/">
-      <img src="images/instagram.png" alt="Facebook">
-    </a></li>
-  </ul>
-  </div>
+      <div class="bg-card">
+        <img src="images/tonic-2.png" alt="Tonic project seakpeak">
+        <h1>Tonic</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
 
-</div>
+      <div class="bg-card">
+        <img src="images/mps-2.png" alt="Tonic project seakpeak">
+        <h1>Multi-Post 
+          <br>Stories</h1>
+        <ul class="categories">
+          <li class="canopy">CANOPY</li>
+          <li>•</li>
+          <li>Back End Dev</li>
+          <li>•</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required</p>
+        <ul class="keyword">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <div class="cta-button">See Project</div>
+      </div>
+    </div> <!-- End of WORKS -->
 
+    <div class="about" id="bg-about">
+      <div class="content">
+        <h1>About <br>Myself</h1>
```

Comment:
- [ ] As mentioned above, we can only have one `<h1>` tag on an HTML webpage. This is because `<h1>` tags are often used in SEO so having many of them on a page makes the site hard to index. I therefore kindly request the use of another heading tag like h2 or h3 to replace the h1.